src/lib/time.js

```diff
@@ -270,25 +275,42 @@ var $builtinmodule = function (name) {
     %Z  Time zone name (no characters if no time zone exists).   
     %%  A literal '%' character.         
     */
-    // accept2dyear
-    mod.accept2year = function accept2year() {
-        throw new Sk.builtin.NotImplementedError("accept2year is not implemented")
-    };
 
-    // strftime()
-    mod.strftime = function strftime() {
-        throw new Sk.builtin.NotImplementedError("strftime is not implemented")
-    };
+    function strftime_f(format, t) {
+        throw new NotImplementedError("time.strftime() is not yet implemented");
+        Sk.builtin.pyCheckArgs("strftime", arguments, 1, 2);
```

Comment:
I disagree.  It depends on how useful the existing code is.  If it's a reasonable start, why make someone else start over from scratch?  As long as there are clear comments about what is there and what is not, I think there are situations where it could be okay...  @Jacco I'm not sure if this applies here or not.